### PR TITLE
add jenkins image PR tester job

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: sjb/generated/ami_build_origin_int_rhel_base.xml [hunk 1]
```diff
@@ -136,6 +136,20 @@ ssh -F ./.config/origin-ci-tool/inventory/.ssh_config -t openshiftdevel &#34;bas
...
+set -o errexit -o nounset -o pipefail -o xtrace
+cd &#34;\${HOME}&#34;
+docker pull registry.access.redhat.com/rhel7
+SCRIPT
+chmod +x &#34;${script}&#34;
+scp -F ./.config/origin-ci-tool/inventory/.ssh_config &#34;${script}&#34; openshiftdevel:&#34;${script}&#34;
+ssh -F ./.config/origin-ci-tool/inventory/.ssh_config -t openshiftdevel &#34;bash -l -c \&#34;timeout 14400 ${script}\&#34;&#34; </command>
+        </hudson.tasks.Shell>
+        <hudson.tasks.Shell>
+          <command>#!/bin/bash
```
Comment (bparees) [inline sjb/generated/ami_build_origin_int_rhel_base.xml]: @stevekuznetsov not sure why i'm picking up this change, maybe someone forgot to generate/push this?

**Comment by bparees** [comment]
@stevekuznetsov ok i think this is ready for your review.

File: sjb/generated/ami_build_origin_int_rhel_base.xml (hunk 1 above)
Comment (stevekuznetsov) [inline sjb/generated/ami_build_origin_int_rhel_base.xml]: You should rebase? I don't get any diff with `generate.sh` on master

Comment (bparees) [inline sjb/generated/ami_build_origin_int_rhel_base.xml]: i did, but let me do it again.

Comment (bparees) [inline sjb/generated/ami_build_origin_int_rhel_base.xml]: yeah still there..

Comment (stevekuznetsov) [inline sjb/generated/ami_build_origin_int_rhel_base.xml]: You're adding the `docker pull registry.access.redhat.com/rhel7` step to the base AMI job in your PR. Is that not intended?

File: sjb/config/test_cases/ami_build_origin_int_rhel_base.yml [hunk 2]
```diff
@@ -54,6 +54,10 @@ actions:
       echo "ETCD_DATA_DIR=/tmp/etcd" >> /etc/environment
       SUDO
   - type: "script"
+    title: "pull down the latest rhel image"
+    script: |-
+      docker pull registry.access.redhat.com/rhel7
+  - type: "script"
```
Comment (bparees) [inline sjb/config/test_cases/ami_build_origin_int_rhel_base.yml]: hm.  well there's also this now.  where did *that* come from...

**Comment by bparees** [comment]
> LGTM -- do you think we need to add in the image builds to the build stage AMI?

it would be preferable....  but honestly those builds do not take that long.  just layering a couple files in.

**Pull request merged**